### PR TITLE
Update daily trading mode

### PR DESCRIPTION
**Pull request by GuillaumeDSM**

### CODE REVIEW

File: tests/Evaluator/Strategies/test_simple_mixed_strategies_evaluator.py [hunk 1]
```diff
@@ -36,28 +36,28 @@ class TestSimpleMixedStrategiesEvaluator(AbstractStrategyTest):
...
 
     @staticmethod
     async def test_slow_uptrend(strategy_tester):
-        await strategy_tester.run_test_slow_uptrend(-10, 0.5)
+        await strategy_tester.run_test_slow_uptrend(-16, 0)
 
     @staticmethod
     async def test_sharp_uptrend(strategy_tester):
-        await strategy_tester.run_test_sharp_uptrend(14.5, 12)
+        await strategy_tester.run_test_sharp_uptrend(8, 9.5)
```
Comment (Herklos) [inline tests/Evaluator/Strategies/test_simple_mixed_strategies_evaluator.py]: 1) Do you now why this two are decreasing ?

File: tests/Evaluator/Strategies/test_simple_mixed_strategies_evaluator.py [hunk 2]
```diff
@@ -36,28 +36,28 @@ class TestSimpleMixedStrategiesEvaluator(AbstractStrategyTest):
...
 
     @staticmethod
     async def test_flat_markets(strategy_tester):
-        await strategy_tester.run_test_flat_markets(-5, 2, -4.5, -13.5)
+        await strategy_tester.run_test_flat_markets(5.5, 3, 2, -2.5)
 
     @staticmethod
     async def test_slow_uptrend(strategy_tester):
-        await strategy_tester.run_test_slow_uptrend(-10, 0.5)
+        await strategy_tester.run_test_slow_uptrend(-16, 0)
```
Comment (Herklos) [inline tests/Evaluator/Strategies/test_simple_mixed_strategies_evaluator.py]: 2) Do you now why this two are decreasing ?

File: tests/Evaluator/Strategies/test_simple_mixed_strategies_evaluator.py (hunk 1 above)
Comment (GuillaumeDSM) [reply tests/Evaluator/Strategies/test_simple_mixed_strategies_evaluator.py]: selling more ealier:
example with 8%
image

Comment (Herklos) [reply tests/Evaluator/Strategies/test_simple_mixed_strategies_evaluator.py]: That's beautiful :+1:

**Pull request merged**